packages/code-snippet-block/src/settings.ts

```diff
@@ -206,56 +206,130 @@ export const settings: BlockSettings = {
             type: 'switch',
             defaultValue: true,
             label: 'Border',
-        },
-        {
-            id: 'border',
-            type: 'multiInput',
-            lastItemFullWidth: true,
-            show: (bundle: Bundle) => bundle.getBlock('withBorder')?.value === true,
-            onChange: (bundle: Bundle): void => {
-                appendUnit(bundle, 'lineWidth');
-            },
-            blocks: [
+            on: [
                 {
-                    id: 'lineStyle',
-                    type: 'dropdown',
-                    defaultValue: 'solid',
-                    choices: [
+                    id: 'border',
+                    type: 'multiInput',
+                    lastItemFullWidth: true,
+                    onChange: (bundle: Bundle): void => {
```

Comment:
```suggestion
                    onChange: (bundle) => {
```
You can do the same everywhere else in the file
Otherwise we have double typing (first with `: BlockSettings` and after with the `(bundle: Bundle): void`